sys-libs/libxcrypt/libxcrypt-4.4.36-r3.ebuild

```diff
@@ -138,9 +138,11 @@ src_configure() {
 	fi
 
 	if use headers-only; then
-		# Nothing is compiled here which would affect the headers for the target.
-		# So forcing CC is sane.
+		# Nothing is compiled which would affect the headers, so we set
+		# CC and PKG_CONFIG to ensure configure passes without defaulting
+		# to the unprefixed host variants e.g. "pkg-config"
 		local -x CC="$(tc-getBUILD_CC)"
+		local -x PKG_CONFIG="none"
```

Comment:
Does configure handle "none" specially, or does it actually try to invoke "none" as a command?

If the latter, please change this to something like "false" so we invoke a command with a known behavior.